src/Features/Core/Portable/Wrapping/SeparatedSyntaxList/AbstractSeparatedSyntaxListWrapper.cs

```diff
@@ -13,24 +14,16 @@ namespace Microsoft.CodeAnalysis.Wrapping.SeparatedSyntaxList
     internal abstract partial class AbstractSeparatedSyntaxListWrapper<
         TListSyntax,
         TListItemSyntax>
-        : AbstractSyntaxWrapper
+        : AbstractSeparatedListWrapper<TListSyntax, TListItemSyntax>
         where TListSyntax : SyntaxNode
         where TListItemSyntax : SyntaxNode
     {
-        //protected abstract string ListName { get; }
-        //protected abstract string ItemNamePlural { get; }
-        //protected abstract string ItemNameSingular { get; }
-
-        protected abstract string Unwrap_list { get; }
-        protected abstract string Wrap_long_list { get; }
 
         protected abstract string Unwrap_and_indent_all_items { get; }
-        protected abstract string Unwrap_all_items { get; }
-        protected abstract string Indent_all_items { get; }
+
         protected abstract string Align_wrapped_items { get; }
         protected abstract string Indent_wrapped_items { get; }
```

Comment:
why were these not moved up?

Reply:
These options aren't offered for the initializer expression, so I left them in the SeparateSyntaxListWrapper. I can move them if it is preferred to have them all in one spot.

Reply:
i'm curious why not?  if they're not appropriate for initializers can we have a comment explaining as such?

Reply:
They support wrapping that, as far as I am aware, isn't applicable to initializers. 

```cs
// Align Wrapped Items
LongFunction("hello",
             "my",
             "name",
             "is",
             "Kris");

// Indent Wrapped Items
LongFunction("hello",
    "my",
    "name",
    "is",
    "Kris");
```

Reply:
And, yea I can add a comment explaining why they are only applicable to this wrapper. Unless those styles need to be supported for initializers too.

Reply:
Comment added.

Reply:
note: a better comment in the code would be to include the information you had in this PR.  i.e. show the example of wrapping we want for arguments that we would not want for an initializer.  Thanks!

Reply:
i don't see the comment added in the code :)